axis.go

```diff
@@ -104,12 +105,25 @@ func makeAxis() (Axis, error) {
 		Scale:   LinearScale{},
 	}
 	a.Label.TextStyle = draw.TextStyle{
-		Color: color.Black,
-		Font:  labelFont,
-	}
-	a.Tick.Label = draw.TextStyle{
-		Color: color.Black,
-		Font:  tickFont,
+		Color:  color.Black,
+		Font:   labelFont,
+		XAlign: -0.5,
+		YAlign: 0,
+	}
+	if vertical {
```

Comment:
I'd probably do it like this, so that only the pieces which are changing conditionally are changed by the conditional.
```
var xalign, yalign = -0.5, 0
if vertical {
  xalign, yalign = -1, -0.5
}
a.Tick.Label = draw.TextStyle{…, XAlign: xalign, YAlign: yalign }
```